src/test/compile-fail/borrowck/borrowck-match-binding-is-assignment.rs

```diff
@@ -8,6 +8,9 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// revisions: ast mir
+//[mir]compfile-flags: -Zemit-end-regions -Zborrowck-mir
```

Comment:
I think you want to use `[mir]compile-flags` here. I think that's why you're getting the travis error.

Reply:
Ack, this is so embarrassing!